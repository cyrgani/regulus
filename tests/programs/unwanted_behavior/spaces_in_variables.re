# TODO: should be a lot of syntax errors
=(a b, 2),
assert_eq(a b, 2),
print(a b),
p r i   n t (   5    ),
a  ssert_eq(2 3, 23),