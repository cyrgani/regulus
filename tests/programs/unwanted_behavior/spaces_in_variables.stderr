
ArgumentError: expected `2` args, found `3` args for `=`